docs/reference/controls/carousel.md

```diff
@@ -10,6 +10,37 @@ The carousel has an items collection, and displays each item as a page, in seque
 
 You can use the carousel control to create a slide show.
 
+## Useful Properties
+
+You will probably use these properties most often:
+
+<table>
+  <thead>
+    <tr>
+      <th width="316">Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>ItemsSource</code></td>
```

Comment:
It might be better off putting this on the `ItemsControl` page and then putting a link to it here instead. These are shared properties with all items controls and it will save someone the trouble of trying to find all references to them if they ever need to updated. But I am also not too sure about this either.

Reply:
There was no "useful properties" section in this page, so I've added the same as the ItemsControl for a bit of standardisation. I find useful for new joiner to have information directly in the page even at the cost of small duplication...but as always It's just a matter of preference and I'm happy to change it if needed

Reply:
Yeah, I am really in two minds about this. The downside with a link is that someone might scroll past it and not find what they are looking for. Ideally you would be able to share the markdown between all the pages from a single source and then have the doc system merge it all together. Apparently it can be done with some plugin for Docusaurus. But I have no idea what's installed on the doc site currently and I can't say I have much experience configuring it either.

Reply:
This is out of scope here, but we are trying to figure out how we could maybe pull these docs from the controls code directly. For now I am happy with wither solution and I'd say let's keep it as it is now.

Reply:
I imagine it's just a matter of enabling [this](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#generatedocumentationfile) and parsing the output somehow. But nice to know that is being considered!